apps/pie-microsite/src/_11ty/shortcodes/tokensTable/index.js

```diff
@@ -77,6 +77,17 @@ const buildColorExample = token => {
     return `<div class="${classes.join(' ')}" style="${cssVariable}";></div>`;
 };
 
+/**
+ * Builds the example spacing swatch to show on the token list item
+ * @param {string} token - the token value i.e. 
+ * @returns {string} - the color swatch example HTML string
+ */
+const buildSpacingExample = token => {
+    const cssVariable = `${token}px`;
```

Comment:
I'd make the name a bit more descriptive here – is this passing over the spacing token value?

If so, `spacingTokenValue` might be a better name? (or whatever you think describes it best!)